pkg/utils/kubernetes/leaderelection.go

```diff
@@ -28,14 +28,14 @@ import (
 // ReadLeaderElectionRecord returns the leader election record for a given lock type and a namespace/name combination.
 func ReadLeaderElectionRecord(ctx context.Context, client client.Client, lock, namespace, name string) (*resourcelock.LeaderElectionRecord, error) {
 	switch lock {
-	case resourcelock.EndpointsResourceLock:
+	case resourcelock.EndpointsLeasesResourceLock:
 		endpoint := &corev1.Endpoints{}
 		if err := client.Get(ctx, Key(namespace, name), endpoint); err != nil {
 			return nil, err
 		}
 		return leaderElectionRecordFromAnnotations(endpoint.Annotations)
 
-	case resourcelock.ConfigMapsResourceLock:
+	case resourcelock.ConfigMapsLeasesResourceLock:
```

Comment:
Instead of using `ConfigMapsLeasesResourceLock` just don't use any constant (since there is none anymore) and write "configmaps" explicitly here.